src/components/OrbSeal.tsx

```diff
@@ -1,8 +1,9 @@
+import { dropShadow, strokeWidth } from 'classnames/tailwind'
 import classNamesToString from 'helpers/classNamesToString'
-import classnames, { strokeWidth } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 
-const mainStroke = classnames(strokeWidth('stroke-1.5'))
+const mainStroke = strokeWidth('stroke-1.5')
+const orbShadow = dropShadow('drop-shadow-accent')
```

Comment:
Removed filter and used `dropShadow`, looks like they're pretty same talking about optimization, but this has less lines :)